--- a/configure.ac
+++ b/configure.ac
@@ -1,5 +1,5 @@
 AC_PREREQ(2.50)
-AC_INIT([jbig2enc], [0.28], [agl@imperialviolet.org], [jbig2enc-0.28],
+AC_INIT([jbig2enc], [0.29], [agl@imperialviolet.org], [jbig2enc-0.29],
 		[https://github.com/agl/jbig2enc])
 AC_CONFIG_MACRO_DIR([m4])
 AM_INIT_AUTOMAKE([-Wall -Werror foreign no-dependencies])
@@ -13,7 +13,7 @@
 
 # Release versioning
 GENERIC_MAJOR_VERSION=0
-GENERIC_MINOR_VERSION=28
+GENERIC_MINOR_VERSION=29
 GENERIC_MICRO_VERSION=0
 
 # API version (often = GENERIC_MAJOR_VERSION.GENERIC_MINOR_VERSION)
